subsys/bluetooth/controller/ll_sw/ull_iso.c

```diff
@@ -1816,10 +1816,19 @@ static isoal_status_t ll_iso_pdu_alloc(struct isoal_pdu_buffer *pdu_buffer)
 	node_tx = ll_iso_tx_mem_acquire();
 	if (!node_tx) {
 		LOG_ERR("Tx Buffer Overflow");
-		/* TODO: Report overflow to HCI and remove assert
+
+		/* Assert if allocation requested in ISR context, i.e. non-HCI packet initiated call
+		 * path.
+		 *
+		 * FIXME: isoal_tx_sdu_fragment() is invoked by ll_iso_transmit_test_send_sdu() in
+		 *        ull_conn_iso_ticker_cb() which executes in ISR context.
+		 * TODO: Handling assertion in Controller implementations using meta-IRQ.
```

Comment:
You replaced 1 TODO with a FIXME and a TODO. That sounds counterproductive :D 

I'm sure that both the FIXME and the TODO text here makes sense to you, but based on the sparse information, I, and probably others, would not enough information to understand what needs to be fixed and/or done. Please elaborate both, create GH issues or fix them in this PR